centos7/Dockerfile.base.centos7

```diff
@@ -0,0 +1,21 @@
+ARG RELVER
+FROM centos:7
```

Comment:
Would it be better to use a more specific tag here?  For instance `centos7:centos7.7.1908`?

Reply:
cc @crunchyjohn @crunchyheath 

From my perspective, it's one more thing to maintain and if we are always getting the latest and greatest CentOS 7 I'd prefer that. However, if we had to peg it at a particular version due to, say avoid an upgrade for a breaking reason, I'd would not be opposed.

However, open to other ideas around this.

Reply:
Yeah, my thoughts are around configuration management and reproducible builds. Luckily, with this refactor, it isolates management of it to a single file.

Reply:
It is now only in one place (base) and the metadata tags the os version as 7.7, so I'd either drop the os metadata for maintenance or be fine with upkeeping both the metadata and specific tag

Reply:
I have no strong opinions on the metadata other than making sure that `docker inspect` can reliably produce the specific version (and other image identification/verification) information.